### ISSUE TITLE
Natural sorting on file names

### ISSUE THREAD

**Issue by wwoods** [guildai/guildai#284]
Using a package like [natsort](https://pypi.org/project/natsort/), it seems that the `guild view` Files tab should sort files with numbers in the file name naturally (such that `file10` comes after `file2`, rather than before).

For our use case anyway, this would also hopefully apply to the arrow-key based navigation when viewing images (we have files named `img1, img2, img3, ..., img9, img10, img11, ...`)

**Comment by gar1t** [comment]
Great idea! This will land in the next pre-release (today or tomorrow).

**Comment by wwoods** [comment]
Thanks again for the quick response! Just installed the master branch -- arrow keys work, looks like "Files" list in browser tab isn't sorted naturally.

**Comment by gar1t** [comment]
Yes, shucks, I see that. The data widget there is applying its own sort over what Guild is doing. Will look to fix.

**Comment by gar1t** [comment]
This should be fixed via c82ceb1. It's not perfect as the table will still sort using lexical ordering when you toggle the sort direction. But the initial view is sorted using natsort and you can return to that order via a third toggle state.
